tests/integration/docusaurus/connecting_to_your_data/in-memory/pandas_python_example.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import pandas as pd
+from ruamel import yaml
+
+import great_expectations as ge
+from great_expectations.core.batch import RuntimeBatchRequest
+
+context = ge.get_context()
+
+datasource_config = {
+    "name": "example_datasource",
+    "class_name": "Datasource",
+    "module_name": "great_expectations.datasource",
+    "execution_engine": {
+        "module_name": "great_expectations.execution_engine",
+        "class_name": "PandasExecutionEngine",
+    },
+    "data_connectors": {
+        "default_runtime_data_connector_name": {
+            "class_name": "RuntimeDataConnector",
+            "module_name": "great_expectations.datasource.data_connector",
+            "batch_identifiers": ["default_identifier_name"],
+        },
+    },
+}
+
+context.test_yaml_config(yaml.dump(datasource_config))
+
+context.add_datasource(**datasource_config)
+
+# creating our example Pandas dataframe
+df = pd.DataFrame(np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]]), columns=["a", "b", "c"])
```

Comment:
We can be even more concise and eliminate the numpy import by doing this:
```suggestion
df = pd.DataFrame([[1, 2, 3], [4, 5, 6], [7, 8, 9]], columns=["a", "b", "c"])
```

Reply:
Love it